File: Testes de Desenvolvimento/Cadastro de Produto/Frontend/Frontend/Frontend/Models/Produto.cs
﻿using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.SqlTypes;
using System.Web.Mvc;

namespace Frontend.Models
{
    public class Produto
    {
        public string Id { get; set; }
        public string Nome { get; set; }
        public string Categoria_Id { get; set; }
        public decimal Preco { get; set; }

        [DisplayName("Categoria")]
        public IEnumerable<SelectListItem> Categoria { get; set; }

    }
}

File: Udemy/Java 2021/Aulas/exercicios/src/streams/Produto.java
package streams;

public class Produto {

	final String nome;
	final double preco;
	final double desconto;
	final boolean temFrete;
	
	public Produto(String nome, double preco, double desconto, boolean temFrete) {
		this.nome = nome;
		this.preco = preco;
		this.desconto = desconto;
		this.temFrete = temFrete;
	}
	
}
